src/components/Input/index.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { letterSpacing, colors, size } from '../../utils'
+
+const { black, gray3 } = colors
+
+const Input = styled.input`
+  width: 100%;
+  padding: 1rem;
+  box-sizing: border-box;
+  border: 1px solid rgba(224, 224, 224, 0.5);
+  background: rgba(196, 196, 196, 0.05);
+  outline: none;
+  font-weight: 700;
+  font-size: ${size.xxs};
+  letter-spacing: ${letterSpacing.medium};
+  font-family: Karu;
+  ::placeholder,
+  ::-webkit-input-placeholder {
+    color: ${black};
+  }
+  :focus {
+    ::placeholder {
+      color: transparent;
+    }
+  }
+`
+const Label = styled.label`
+  width: 100%;
+  text-transform: uppercase;
+  font-family: Karu;
+  font-weight: 550;
+  font-size: ${size.xxs};
+  letter-spacing: ${letterSpacing.medium};
+  color: ${gray3};
+`
+const Textarea = styled.textarea`
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  padding: 1rem;
+  border: 1px solid rgba(224, 224, 224, 0.5);
+  resize: none;
+  background: rgba(196, 196, 196, 0.05);
+  outline: none;
+  font-weight: 550;
+  font-size: ${size.xxs};
+  letter-spacing: ${letterSpacing.medium};
+  font-family: Karu;
+  ::placeholder,
+  ::-webkit-input-placeholder {
+    color: ${black};
+  }
+  :focus {
+    ::placeholder {
+      color: transparent;
+    }
+  }
+`
+
+const InputContainer = styled.div`
+  width: 45%;
+  margin: 10px 0;
+`
+const TextareaContainer = styled.div`
+  width: 100%;
+  height: 230px;
+  margin: 10px 0;
+  @media (max-width: 800px) {
+    height: 180px;
+  }
```

Comment:
I get why you did this:
1. The **input container** for putting two inputs next to each other when using flexbox but that's not the component job its, the one who uses it should make containers for them and in this way, if I want to use one input it won't take 100% of its parent and that's not wanted in my opinion

2. The **text area container** is to make get the exact height like the Figma but you can do that in two different ways, **one**: set the height in the text area component itself, **two**: give rows and cols to the text area element https://www.w3schools.com/tags/att_textarea_rows.asp

Reply:
no not even close : 
1- the input container is for input itself, I just, in the storybook, import two times and add it to div that has flex display just to show that the width is fine,
 
2- yes you are alright  I can add a column to textarea but I beleive the percentage will guarantee the flexibility

Reply:
@fadeomar read again please this is wrong it doesn't make sense